docs/openapi/components/schemas/credentials/CommercialInvoiceCredential.yml

```diff
@@ -95,7 +95,278 @@ properties:
       - type
       - name
   credentialSubject:
-    $ref: ../common/Invoice.yml
+    title: Invoice
```

Comment:
Why are you defining this inline instead of modifying the existing common object? 
Doing it this way will reduce reusability.

Reply:
This is with respect to issue: https://github.com/w3c-ccg/traceability-vocab/issues/624. Pulling in the entire object can pull in a lot of attributes that the issuer is not interested in. This narrows down the scope to only use the attributes required by the document the credential is replacing.